pkg/wasm/exports.go

```diff
@@ -0,0 +1,68 @@
+//go:build tinygo
+
+package main
+
+// This file contains Go/WASM functions exported to and called by JS.
+
+// alloc pre-allocates memory used in string parameter passing.
+//
+//export alloc
+func alloc(size uint32) *byte {
+	buf := make([]byte, size)
+	return &buf[0]
+}
+
+// stop is called when JS/UI interactions triggers stop
+//export stop
```

Comment:
When I save this file in my editor, it runs `goimports` and that adds a blank comment line before `//export stop`, and all the other comments with exports. What command do you run on save, and is it up-to-date? (there were recent changes with the updated go doc changes for 1.19 that reformatted comments, so you need to be up-to-date with your installed goimports.

It seems there is not a make recipe for formatting the Go source in `make tidy`, so CI passes even if code is not formatted.

I install it with:
```
go install golang.org/x/tools/cmd/goimports@master
```

Reply:
i think I use gofumports, i will investigate.

Reply:
ok, I will investigate. it should be made part of local tooling with local GOBIN and part of the build process, do you agree?

Reply:
ok. gofumpt adds this too. The linter should have found this. that's sad :(